src/program.ts

```diff
@@ -1044,7 +1044,9 @@ export class Program extends DiagnosticEmitter {
       // TODO: for (let [alias, name] of globalAliases) {
       for (let _keys = Map_keys(globalAliases), i = 0, k = _keys.length; i < k; ++i) {
         let alias = unchecked(_keys[i]);
-        let name = assert(globalAliases.get(alias));
+        let name = globalAliases.get(alias);
```

Comment:
Can just use `changetype<Map<string,string>>(globalAliases.get(alias))` since the value can't be `null`

Reply:
but it can be undefined right?
```typescript
alias = "bad key"
let name = changetype<string>(globalAliases.get(alias))
if (!name.length) continue; // explicitly disabled
```
throws
```
ERROR: TypeError: Cannot read property 'length' of undefined
```
I suppose that's not really much different than the assertion though and shouldn't really happen anyway

Reply:
Hmm, yeah, that's strange. Might be `cli/asc.js` wrongly inserting an `undefined` into the map.

Reply:
Btw, `alias = "bad key"` doesn't happen since we `_keys = Map_keys(globalAliases)`

Reply:
Ah okay- well in that case maybe we should just use `!` and ignore the undefined case